### PR TITLE
Install library with exe permissions. JB#51013

### PR DESCRIPTION
**Pull request by rinigus**
Similar to other libs, install using 755

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -145,7 +145,7 @@ INSTALL_FILES = $(INSTALL) -m 644
 INSTALL_PLUGIN_DIR = $(DESTDIR)$(ABS_PLUGIN_DIR)
 
 install: $(INSTALL_PLUGIN_DIR)
-	$(INSTALL_FILES) $(RELEASE_LIB) $(INSTALL_PLUGIN_DIR)
+	$(INSTALL) -m 755 $(RELEASE_LIB) $(INSTALL_PLUGIN_DIR)
```
Comment (monich) [inline Makefile:147]: I think that was the only use of INSTALL_FILES macro, it can be removed then.

**Comment by monich** [comment]
Note that it's not a general-purpose lib, it's a plugin. Is it necessary to make those executable too?

**Comment by rinigus** [comment]
> Note that it's not a general-purpose lib, it's a plugin. Is it necessary to make those executable too?

I know. On Fedora, anything that is packaged as binaries (not noarch), is expected to have debug symbols. To search for debug symbols, it checks executable files only. As a result, even in the case of plugin, it is expected to have 755 permissions as any other loadable lib. Hence the change

File: Makefile (hunk 1 above)
Comment (rinigus) [reply Makefile:147]: I dropped it in the revised commit. Please review

**Review by monich — APPROVED** [review]
Silly Fedora )) OK, merging!

**Comment by rinigus** [comment]
Thanks! :)

**Pull request merged**